src/NumericInterpolator/NSOrdinalScale.class.st

```diff
@@ -39,7 +39,15 @@ NSOrdinalScale >> initialize [
 
 { #category : #transformation }
 NSOrdinalScale >> invert: anObject [
-	self shouldNotImplement
+
+	^ NSScale linear
+		  domain: {
+				  domain first.
+				  domain last };
+		  range: {
+				  range first.
+				  range last };
+		  invert: anObject
```

Comment:
But this is not the same implementation as `scale:`? Should not range and domains be inverted here?

Reply:
in this case he is using `invert:`

Reply:
we are creating a linear scale with the limits of the domain and range and then we return the invert the number. It is not the same as the `scale:`, there they scale the method.

Reply:
Ohhh yes!!! Stupid rasta!